build_tools/cmake/test.sh

```diff
@@ -28,6 +28,8 @@ export CTEST_PARALLEL_LEVEL=${CTEST_PARALLEL_LEVEL:-$(nproc)}
 # and turning on the llvmaot ones.
 export IREE_VULKAN_DISABLE=${IREE_VULKAN_DISABLE:-0}
 export IREE_LLVMAOT_DISABLE=${IREE_LLVMAOT_DISABLE:-0}
+# CUDA is off by default.
+export IREE_CUDA_DISABLE=${IREE_VULKAN_DISABLE:-1}
```

Comment:
Use `IREE_CUDA_DISABLE` instead here?
```suggestion
export IREE_CUDA_DISABLE=${IREE_CUDA_DISABLE:-1}
```

Reply:
Yes...